spec/controllers/legacy_users_controller_spec.rb

```diff
@@ -1,17 +0,0 @@
-require 'spec_helper'
-
-describe UsersController, :type => :controller do
```

Comment:
We are really bad at managing specs in our test suite. We have three seperate users controller tests, which shouldn't be the case. I consolidated them into one users_controller_spec.rb.

We are testing the wrong things in certain tests, we have like integration tests in /tasks, /forms, etc, we aren't naming the test files the correct way, etc etc.